%YAML:1.0

#common parameters

imu_topic: "/camera/gyro_accel/sample"
image_topic: "/camera/color/image_raw"
depth_topic: "/camera/depth/image_raw"
need_compress: 1 # 0 for subscriping compressed image  

result_path: "/home/fjg/work/catkin_dio/catkin_dio/result/"
name: kinect_dk



camera_name: camera
image_width: 800
image_height: 600


#distortion_parameters:
k1: 0
k2: 0
p1: 0
p2: 0

#projection_parameters:
fx: 712.103
fy: 712.07
cx: 400.079
cy: 299.316


extrinsicRotation: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [1.0, 0.0, 0.0,
          0.0, 1.0, 0.0,
          0.0, 0.0, 1.0]    
#Translation from camera frame to imu frame, imu^T_cam
extrinsicTranslation: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [0.0, 0.0, 0.0]


#Volume parameters

voxel_size: 10
truncated_size: 60
voxel_x: 600
voxel_y: 500
voxel_z: 800 

init_x: 0
init_y: 0
init_z: 1000

#Controller parameters
render_surface: 1
PST_path: "/home/fjg/work/catkin_dio/catkin_dio/src/PST/"
save_trajectory: 1
save_scene: 1

max_iteration: 18
scaling_coefficient1: 0.25
scaling_coefficient2: 0.15
init_fitness: 0.6
momentum: 0.9
scaling_inherit_directly: 0



